gloo/transport/tcp/tls/openssl/openssl.cc

```diff
@@ -0,0 +1,111 @@
+#include <openssl/err.h>
+#include <openssl/ssl.h>
+
+#include "dynamic_library.h"
+
+namespace {
+DynamicLibrary libssl("libssl.so", "libssl.so.1.1");
```

Comment:
Please avoid global constructors, as they are called at program/library load time.
Instead static variables inside functions which are constructed during 1st call to the function.
```suggestion
DynamicLibrary& get_libssl() {
  static DynamicLibrary libssl("libssl.so", "libssl.so.1.1");
  return libssl;
}

```

Reply:
addressed